src/platform/forms-system/src/js/review/ReviewChapters.jsx

```diff
@@ -32,8 +33,18 @@ class ReviewChapters extends React.Component {
   handleToggleChapter({ name, open, pageKeys }) {
     if (open) {
       this.props.closeReviewChapter(name, pageKeys);
+      if (this.props.form.formId === '22-1995') {
```

Comment:
Hard-coding logic for a specific form isn't an approach we can accept into the forms library. Please find another way to accomplish this. I recommend either looking into callbacks we can pass from the form config, or having a conversation with the analytics team to determine if this is useful information to gather for all forms.